### ISSUE TITLE
tvg-logo uses random local address instead of server IP

### ISSUE THREAD

**Issue by troykelly** [xteve-project/xTeVe#300]
#### Please first read the [documentation](https://github.com/xteve-project/xTeVe-Wiki/blob/master/en/configuration.md). Problems affecting the client, e.g. Plex or Emby should be reported there in the forum.

**Describe the bug**
xteve is randomly selecting an interface to use for `tvg-logo`

**Server (please complete the following information):**
- OS Linux Ubuntu
- Arch Intel 64
- xTeVe Version and Build 2.2.0 Build: 0200

**To Reproduce**
1. Use xteve
2. Look at m3u

**Expected behavior**
The correct server IP should be selected, or the interface should be selectable via configuration

**Screenshots**
```text
#EXTINF:0 channelID="x-ID.57" tvg-chno="305" tvg-name="SBS PopDesi" tvg-id="305" tvg-logo="http://172.27.0.1:34400/images/771d895da337f257b1a690df227a877c.png" group-title="FTA",SBS PopDesi
http://broadcast.example.com/stream/771d895da337f257b1a690df227a877c
#EXTINF:0 channelID="x-ID.58" tvg-chno="306" tvg-name="SBS Chill" tvg-id="306" tvg-logo="http://172.27.0.1:34400/images/251973bc29a99699d26fa4eb7d8b349e.png" group-title="FTA",SBS Chill
http://broadcast.example.com/stream/2452c05838b36ca2a82617d7d446b9df
#EXTINF:0 channelID="x-ID.59" tvg-chno="307" tvg-name="SBS PopAsia" tvg-id="307" tvg-logo="http://172.27.0.1:34400/images/02b046550afa6897c74b7b8169a63621.png" group-title="FTA",SBS PopAsia
http://broadcast.example.com/stream/463caa533fd8df97214a31f237a626de
```
**Desktop (please complete the following information):**
 - OS: N/A
 - Browser N/A
 - Version N/A

**Additional context**

**Comment by dbowles1975** [comment]
I saw something similar once when installed on a Windows PC running Docker. It wanted to grab the WSL network address for some reason. I had to shut down all other NICs then reinstall.

**Comment by jlafitte** [comment]
Did you ever figure out a fix for this?  Same thing happened to me.  I believe it happens after the scheduled update.  I'm not sure how to fix, but the workaround temporarily is to go to the playlist source and update through the web interface.

**Comment by troykelly** [comment]
Unfortunately no.

**Comment by dbowles1975** [comment]
> Did you ever figure out a fix for this? Same thing happened to me. I believe it happens after the scheduled update. I'm not sure how to fix, but the workaround temporarily is to go to the playlist source and update through the web interface.

There's a fork that allows you to enter the host IP. This prevents xTeVe from grabbing other network adapters. The dev reached out to me after a post on another forum. I've been running it for months now with no issue. 

https://github.com/SCP002

**Comment by troykelly** [comment]
> There's a fork that allows you to enter the host IP. This prevents xTeVe from grabbing other network adapters. The dev reached out to me after a post on another forum. I've been running it for months now with no issue.

@dbowles1975 
Can you enter the host name (rather than host IP)?

**Comment by dbowles1975** [comment]
> > There's a fork that allows you to enter the host IP. This prevents xTeVe from grabbing other network adapters. The dev reached out to me after a post on another forum. I've been running it for months now with no issue.
> 
> @dbowles1975 Can you enter the host name (rather than host IP)?

Nope.

**Comment by troykelly** [comment]
Yeah - that's been my biggest frustration - lack of support for SSL (via a reverse proxy) and correct configuration (or honouring) of a hostname
